### PR TITLE
promote e2e should wait gracefully terminated pod until preStop hook completes the process to conformance

### PR DESCRIPTION
**Pull request by pontiyaraja**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind api-change
> /kind bug
>
/kind cleanup
> /kind design
> /kind documentation
> /kind failing-test
> /kind feature
> /kind flake

**What this PR does / why we need it**:
promote e2e should wait gracefully terminated pod until preStop hook completes the process to conformance
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:
This PR to promote e2e should wait gracefully terminated pod until preStop hook completes the process to conformance
**Does this PR introduce a user-facing change?**:

```release-note
NONE
```
/area conformance
/area testing
@kubernetes/cncf-conformance-wg 
@spiffxp @brahmaroutu

### CODE REVIEW

File: test/e2e/node/pre_stop.go [hunk 1]
```diff
@@ -177,7 +177,12 @@ var _ = SIGDescribe("PreStop", func() {
 		testPreStop(f.ClientSet, f.Namespace.Name)
 	})
 
-	It("graceful pod terminated should wait until preStop hook completes the process", func() {
+	/*
+		Release : v1.15
+		Testname: Pods, prestop hook with graceful termination
+		Description: Create a pod with prestop lifecycle, terminate the pod with graceful period seconds. Wait up to the graceful termination period seconds, then verify the pod is in running state after the graceful termination period seconds.
+	*/
+	framework.ConformanceIt("should wait gracefully terminated pod until preStop hook completes the process", func() {
```
[the remark this answers is not in the document]
Comment (pontiyaraja) [reply test/e2e/node/pre_stop.go]: graceful

[the remark this answers is not in the document]
Comment (pontiyaraja) [reply test/e2e/node/pre_stop.go]: its taking ~70 seconds. 5 mins is slow https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/e2e-tests.md#kinds-of-tests

[the remark this answers is not in the document]
Comment (neolit123) [reply test/e2e/node/pre_stop.go]: thanks for providing the value,

**Comment by spiffxp** [comment]
/close
I don't think that we should promote this to Conformance:
- the directory its in claims everything in there is legacy and should finds its way to test/e2e/common or test/e2e_node
- it's verifying that graceful termination should take place after prestop by... using a prestop hook that never finishes? so it's actually verifying that graceful termination doesn't take place if prestop doesn't finish, which is not quite the same thing
- its trying to talk via proxy to a kubelet to list pods on a node, vs. asking the apiserver what pods are on a node (it assumes the kubelet is listening on its default port)

**Pull request closed without merging**